config/services/front/services_prod.yml

```diff
@@ -49,6 +49,9 @@ services:
 
     prestashop.adapter.module.repository.module_repository:
       class: 'PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository'
+      arguments:
+        - '@=service("prestashop.adapter.legacy.configuration").get("_PS_ROOT_DIR_")'
+        - '@=service("prestashop.adapter.legacy.configuration").get("_PS_MODULE_DIR_")'
```

Comment:
Using `get()` without shop constraint is deprecated since 1.7.8.0

Reply:
I check in `\PrestaShop\PrestaShop\Adapter\Configuration`. I didn't see anything in the class about a deprecation and nothing in devdocs.

Reply:
Look at the `get()` method, if no `$shopConstraint` is passed, the method `buildShopConstraintFromContext()` is called (https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Configuration.php#L106) and triggers a deprecated https://github.com/PrestaShop/PrestaShop/blob/1b82184de2242cff69796fb5ceae35e7d5ee67bd/src/Adapter/Configuration.php#L424-L427

Reply:
Oh shit. Bad read my code, all my apologies.

Reply:
Fixed. I remove the call to `prestashop.adapter.legacy.configuration`. `_PS_ROOT_DIR_` &`_PS_MODULE_DIR_` are not constants, not configurations.